### DISCUSSION TITLE
Running with spyserver, sample rate way low

### DISCUSSION THREAD

**Discussion by rough316** [AlexandreRouma/SDRPlusPlus#831] [General]
I am running an Airspy HF Discovery on a pi, when I connect to that spyserver via SDRPP the sample rates shown are 14.250k, then 7.125, and some other lower sample rate. I can hear audio just fine, but would like to see more than 14khz of spectrum. 

Is this possible? With SDR# I get the full 768,000 or BW. 

Tried the official and nightly build, both act the same. 

Would like to solve this, would be great to have SDRPP running on my Mac.

**Comment by gvanem** [comment]
Is your Airspy HF on the same LAN? Not sure if this is the issue I once had. Alexandre explained it [here](https://github.com/AlexandreRouma/SDRPlusPlus/issues/622):

> SDR# doesn't run full IQ, it runs FFT&VFO and has some features enabled to make 8bit pcm work.

**Comment by AlexandreRouma** [comment]
SDR++ cannot use SpyServer's VFO+FFT mode and your spyserver is likely configured to limit the maximum IQ bandwidth it sends.

**Reply by rough316** [reply]
Thank you. You are correct there was a limit in there. Is there some alternative to spyserver where I can use SDRPP and get a view of more spectrum?

**Reply by AlexandreRouma** [reply]
SDR++ server is built in, has better performance and works with any SDR that SDR++ supports. simply start SDR++ with the `--server` parameter. Make sure you are running the nightly version on  boht the server and client. In server mode, it can run headless, meaning you can run it on a server

**Reply by rough316** [reply]
Thank you, I am going to try this out.

**Reply by rough316** [reply]
Got it built both on pi3b and on Macbook Pro M1, works very good. You have done amazing work with this software. Thank you!
